### PR TITLE
Vaishnavi_Assignment_27_35Min_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
image

### CODE REVIEW

File: src/vaishnaviVJun20_arrayExamples/MeetingRoom.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		meetingRoom.bookMeetingRoom();
+		System.out.println("Which Meeting room would you like to book : Takshashila/AgraFort/PratapGadh ? ");
+		String meetingRoomName = scanner.next();
+		meetingRoom.bookMeetingRoom(meetingRoomName);
+		System.out.println("Which Meeting room would you like to book and for how many hours : Takshashila/AgraFort/PratapGadh/Nalanda ? ");
+		meetingRoomName = scanner.next();
+		int hrs = scanner.nextInt();
+		meetingRoom.bookMeetingRoom(meetingRoomName, hrs);
+		meetingRoom.displayAvilableMeetingRoomForBooking();
+		scanner.close();
```
Comment (harshvegada) [inline src/vaishnaviVJun20_arrayExamples/MeetingRoom.java]: Hi @VaishnaviVJun20,
Here while calling method you are printing time for booked room but you need to check which for room booked for how many hours, this FR not fulfilled in code

what you should do is define String array named as meetingRoomName and second array which contains hours.
when someone booked room that time you should update the same index value.

String[] meetingRoomName = {"Nalanda", "Takshashila", "AgraFort","PratapGadh"};
int hours[] = {0,0,0,0};

this hint may help you to think on this. for now we know array which can store multiple value, so think on this

**Review by harshvegada — COMMENTED** [review]
work on comment

File: src/vaishnaviVJun20_arrayExamples/MeetingRoom.java (hunk 1 above)
Comment (VaishnaviVJun20) [reply src/vaishnaviVJun20_arrayExamples/MeetingRoom.java]: -- Updated the code, Please review

File: src/vaishnaviVJun20_arrayExamples/MeetingRoom.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+				takshashilaHours = hours;
+			} else
+				System.out.println(meetingRoomName + " meeting room is already booked for " + hours + "hr.");
+		} else if (meetingRoomName == MEETINGROOMS.AgraFort) {
+			if (bookingStatusForAgraFort == false) {
+				System.out.println(meetingRoomName + " meeting room is booked for " + hours + "hr.");
+				bookingStatusForAgraFort = true;
+				agraFortHours = hours;
+			} else
+				System.out.println(meetingRoomName + " meeting room is already booked for " + hours + "hr.");
```
Comment (harshvegada) [inline src/vaishnaviVJun20_arrayExamples/MeetingRoom.java:62]: Hi @VaishnaviVJun20 
if you check this line gives wrong information, line 48, 55 and 62
this gives wrong informart

**Review by harshvegada — APPROVED** [review]
looks rest thing good work in

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**